gapic/src/main/com/google/gapid/models/Profile.java

```diff
@@ -202,10 +202,10 @@ public TimeSpan getSlicesTimeSpan() {
       return new TimeSpan(start, end);
     }
 
-    public Double getGpuPerformance(List<Long> commandIndex, int metricId) {
+    public Service.ProfilingData.GpuCounters.Perf getGpuPerformance(List<Long> commandIndex, int metricId) {
       CommandIndex indexStr = new CommandIndex(commandIndex);
-      Map<Integer, Double> metrics = perfLookup.get(indexStr);
-      return (metrics == null) ? Double.NaN : metrics.getOrDefault(metricId, Double.NaN);
+      Map<Integer, Service.ProfilingData.GpuCounters.Perf> perfs = perfLookup.get(indexStr);
+      return (perfs == null) ? null : perfs.getOrDefault(metricId, null);
```

Comment:
`null` is already the default, so using `getOrDefault` here only adds additional overhead if `null` is returned.

Reply:
Indeed! Updated accordingly.